### ISSUE TITLE
paper

### ISSUE THREAD

**Issue by yezer24** [eric8607242/NAS_with_GNN_predictor#1]
Hello,do you have a paper about this code? I want to read it,,thank you.

**Comment by eric8607242** [comment]
Hi, this project is only a course project. There is no paper about this code.
The pipeline of this code can be summarized as follows:
1. Define the search space (In the project, I created the search space based on DARTs, but smaller).
2. Random lots of architectures from the search space and train them from scratch on CIFAR100 to construct a dataset.
3. Split the dataset into a training set and validation set.
4. Training a GCN predictor as a regression task. (An architecture in the search space can be seen as a graph. And the GCN aims to predict the top1 accuracy given the architecture (or graph) as input).
5. Incorporate the pre-trained GCN predictor with evolution to search for the best architecture. (Note that the searching process can be done in seconds because of the pre-trained GCN).

However, the follows are the paper related to GNN predictor-based NAS.
1. BRP-NAS: Prediction-based NAS using GCNs. NIPS 2020.
2. Neural Architecture Performance Prediction Using Graph Neural Networks. DAGM GCPR 2020.
3. A Generic Graph-based Neural Architecture Encoding Scheme for Predictor-based NAS. ECCV 2020. 

Hopes these help you : ).

**Comment by yezer24** [comment]
Thank you very much!
